lib/cpp/src/thrift/transport/TPipeServer.cpp

```diff
@@ -400,13 +395,12 @@ bool TAnonPipeServer::createAnonPipe() {
   SECURITY_ATTRIBUTES sa;
   SECURITY_DESCRIPTOR sd; // security information for pipes
 
-  if (!InitializeSecurityDescriptor(&sd, SECURITY_DESCRIPTOR_REVISION))
-  {
-    GlobalOutput.perror("TPipeServer InitializeSecurityDescriptor (anon) failed, GLE=", GetLastError());
+  if (!InitializeSecurityDescriptor(&sd, SECURITY_DESCRIPTOR_REVISION)) {
+    GlobalOutput.perror("TPipeServer InitializeSecurityDescriptor (anon) failed, GLE=",
+                        GetLastError());
     return false;
   }
-  if (!SetSecurityDescriptorDacl(&sd, true, NULL, false))
-  {
+  if (!SetSecurityDescriptorDacl(&sd, true, NULL, false)) {
```

Comment:
nit: remove brace line change.

Reply:
This is now inline with the clang format file in the repo. As it's a step in the right direction and doesn't make the PR unclear I think it's fine to leave the formatting change as is.